CLAUDE.md

```
@@ -0,0 +1,448 @@
+- `NEXT_PUBLIC_FLAG_SPOTIFY`: Enable Spotify integration
+- `NEXT_PUBLIC_FLAG_ANALYTICS`: Enable analytics
+- `NEXT_PUBLIC_FLAG_GUESTBOOK_NOTIFICATION`: Enable notifications
+- `NEXT_PUBLIC_FLAG_LIKE_BUTTON`: Enable like buttons
+
+## Pull Request Guidelines
+
+### PR Title Format
+
+Follow Conventional Commits:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)